defaults/main.yml

```diff
@@ -190,4 +190,7 @@ superset_apt_dependencies_packages: ["build-essential", "libssl-dev", "libffi-de
 superset_pip_dependencies_packages: ["setuptools", "psycopg2-binary", "mysql", "Flask-OAuthlib", "crontab"]
 
 # Roles created by superset
-superset_robot_permission_roles: ['Public', 'Gamma', 'Alpha', 'Admin', 'sql_lab']
\ No newline at end of file
+superset_robot_permission_roles: ['Public', 'Gamma', 'Alpha', 'Admin', 'sql_lab']
+
+# Configure superset dashboard position json data limit
+superset_dashboard_position_data_limit: 131072
```

Comment:
cool, we can probably increase this to 16777216 without any problem (since that'll give us an extra byte) but let's leave like this for now since we don't know how that'll affect other parts of the system

Reply:
hmm, wait actually this is a default? I think we should leave this at it's usually value and only override in our own playbooks

Reply:
Okay. I'll confirm with the data team if that will solve it. If not, I will default to that